cmd/dispenser/server.go

```diff
@@ -71,14 +71,19 @@ const topPageTemplate = `
     <script
       src="https://code.jquery.com/jquery-3.3.1.min.js"
       integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
-      crossorigin="anonymous"></script>
-
-    <script>
+	   crossorigin="anonymous"></script>
+	  
+	 <script>
+	  function loadparm(){
+				const queryString = window.location.search;
+				const urlParams = new URLSearchParams(queryString);
+				$('#target').html(urlParams.get('account'));
```

Comment:
This introduces an [XSS vulnerability](https://portswigger.net/web-security/cross-site-scripting). Instead of calling `$('#target').html`, please use something like `$('#target').val`, which is safe to call with this untrusted parameter.

Reply:
$('#target').val is working! and inserted. Thanks for the feedback.

Reply:
changed